2.10/2.10.25/index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<title>Title</title>
+</head>
+<body>
+### 2.10.25
+Человек вводит время, чтобы поставить будильник.
+Выводится сообщение для ввода часов: "Введите часы"
+Потом для ввода минут: "Введите минуты"
+Есть условия у этого будильника:
+- значения часов может быть 0 - 23
+- значения минут может быть 0 - 59
+- будильник можно ставить только утром с 5:00 до 8:00 включительно
+
+Если часы введены неверно, тогда сразу появляется алерт "Неправильные данные" и сообщение для минут не появится.
+И если все условия выполняются, тогда вывести алерт "Будильник готов", иначе "Неправильные данные"
+<script>
+	const hours = +prompt('Введите часы');
+	let minutes;
+
+	if (hours >= 0 && hours <= 23) {
+		minutes = +prompt('Введите минуты');
+
+		if ((hours < 8 && hours >= 5) && (minutes >= 0 && minutes <= 59) || (hours === 8 && minutes === 0)) {
```

Comment:
Let's swap conditions from `(hours < 8 && hours >= 5)` to `(hours >= 5 && hours < 8)`
this is a very minor issue, but it will be great if you fix it :) in order to have your code more readable